<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zInterpolate</class>
 <widget class="QWidget" name="zInterpolate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Region of Interest selections:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Target (select the ROI which Z-coordinates are to be modified):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_target"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Reference (select the ROI which Z-coordinates are to be used for the interpolation reference):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_reference"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Interpolation method:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_interpMethod"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_performInterp">
       <property name="text">
        <string>Perform interpolation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_instruction">
       <property name="text">
        <string>Show instructions</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
